src/index.html

```diff
@@ -318,4 +318,5 @@ <h2>INSTRUCTIONS</h2>
       </section>
     </main>
   </body>
+  <script src="./scripts.js"></script>
```

Comment:
I don't think this is needed when using webpack. Try removing it and see if the application still runs as expected with `npm run start`. I could totally be wrong though 🙂